mdoc/.gitignore

```diff
@@ -1,5 +1,6 @@
-/Test/en.*/
-/Test/html.*/
+#/Test/en.*/
```

Comment:
I believe these changes are not necessary ... can you revert this?

Reply:
This is just temporary. I made this change so that I can edit expected XMLs inside VS. I will revert it back after I am done with all the integration test changes.

Reply:
Already changed .gitignore back

Reply:
Understood ... so with these latest changes, does it mean the PR is ready for the final review/merge?

Reply:
Yes, this PR is ready for final review and merge